README.md

```diff
@@ -129,19 +129,24 @@ The are numbers of ways you can run Unleash.
 
 ### Run from Docker
 
-1. Create a network by running docker network create unleash
+1. Create a network by running the following command:
+
+```sh
+docker network create unleash
+```
+
 2. Start a postgres database:
 
 ```sh
-docker run -e POSTGRES_PASSWORD=some_password \
+docker run -d -e POSTGRES_PASSWORD=some_password \
   -e POSTGRES_USER=unleash_user -e POSTGRES_DB=unleash \
   --network unleash --name postgres postgres
 ```
 
 3. Start Unleash via docker:
 
 ```sh
-docker run -p 4242:4242 \
+docker run -d -p 4242:4242 \
```

Comment:
As above, we'd like to keep readme sample commands blocking. See the general comment for more on that 😄 

```suggestion
docker run -p 4242:4242 \
```